integration-job/config.yaml

```diff
@@ -0,0 +1,19 @@
+presubmits:
+  kyma-project/kyma:
+  - name: kyma-integration
+    trigger: "(?m)^/test kyma-integration"
+    rerun_command: "/test kyma-integration"
+    context: kyma-integration
+    always_run: true
+    skip_report: false
+    agent: kubernetes
+    spec:
+      containers:
+      - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.1
+        volumeMounts:
```

Comment:
can we use presets for injecting service account?
see this https://github.com/knative/test-infra/blob/master/ci/prow/config.yaml#L73